app/mailers/cards_mailer.rb

```diff
@@ -0,0 +1,7 @@
+class CardsMailer < ActionMailer::Base
+  default from: ENV['DEFAULT_EMAIL_FROM_CARDS']
+
+  def pending_cards_notification(email)
+    mail(to: email, subject: 'Наступила даты пересмотра карточек.')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.